subworkflows/local/vcf_sites_extract_bcftools/main.nf

```diff
@@ -27,6 +28,10 @@ workflow VCF_SITES_EXTRACT_BCFTOOLS {
     BCFTOOLS_QUERY(ch_panel_sites, [], [], [])
     ch_versions = ch_versions.mix(BCFTOOLS_QUERY.out.versions.first())
 
+    // Convert TSC to Stitch format ","" to "\t"
+    GAWK(BCFTOOLS_QUERY.out.output, [])
+    ch_versions = ch_versions.mix(GAWK.out.versions)
```

Comment:
This is ok but I am addressing it in a different way, as we are going to use as input a single `--posfile`, therefore, the changes need to be in stitch. I am working on this in my current PR.

Reply:
Okay 👍